pkg/server/datastore/sqlstore/sqlstore.go

```diff
@@ -524,6 +525,12 @@ func (ds *Plugin) Configure(hclConfiguration string) error {
 		return err
 	}
 
+	if err := ds.withWriteTx(context.Background(), func(tx *gorm.DB) (err error) {
```

Comment:
We should change Configure to take a context and pass that here (will require threading through the context from the server catalog code).